extensions/adobe/experience/offer-management/content/image-content.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "meta:license": [
+    "Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$id":
+    "https://ns.adobe.com/experience/offer-management/content/text-content",
+  "type": "object",
+  "title": "Plain Text Content",
+  "description": "Offer Representation",
+  "definitions": {
+    "image-content": {
+      "type": "object",
+      "title": "Image",
+      "description": "Offer image",
+      "properties": {
+        "@type": {
+          "type": "string",
+          "const": "https://ns.adobe.com/experience/offer-management/content/image-content"
+        },
+        "image": {
+          "type": "string",
+          "format": "uri",
+          "title": "Image URL",
+          "description": "Image representing the offer"
+        },
+        "imageName": {
+          "type": "string",
+          "title": "Image Name",
+          "description": "Image name"
```

Comment:
What's the point of the image name?

Reply:
Can be used either as an `alt` or as an image description. Indexable too.